yt_dlp/extractor/syvdk.py

```diff
@@ -10,9 +10,10 @@ class SYVDKIE(InfoExtractor):
         'md5': '429ce5a423dd4b1e1d0bf3a569558089',
         'info_dict': {
             'id': '12215',
+            'display_id': 'isabella-arendt-stiller-op-for-de-konservative-2',
             'ext': 'mp3',
             'title': 'Isabella Arendt stiller op for De Konservative',
-            'description': 'Tidligere formand for Kristendemokraterne Isabella Arendt stiller op til folketingsvalget for Det Konservative Folkeparti. Men hvorfor netop de konservative? og hvordan harmonisere det med hendes ønske om en mere human udlændinge- og integrationspolitikken. Hun er med fra start og fortæller hvorfor. <br /><br />Vi har også LOKK - Landsorganisation af Kvindekrisecentre’s formand Karin Gaardsted i programmet. Vi får talt om handleplanen mod partnerdrab. Og om der behov for at ændre den generelle samtale om vold i nære relationer og kvindedrab.'
+            'description': 'Tidligere formand for Kristendemokraterne Isabella Arendt stiller op til folketingsvalget for Det Konservative Folkeparti. Men hvorfor netop de konservative? og hvordan harmonisere det med hendes ønske om en mere human udlændinge- og integrationspolitikken.  Hun er med fra start og fortæller hvorfor. <br /><br />Vi har også LOKK - Landsorganisation af Kvindekrisecentre’s formand Karin Gaardsted i programmet. Vi får talt om handleplanen mod partnerdrab. Og om der behov for at ændre den generelle samtale om vold i nære relationer og kvindedrab.'
```

Comment:
I think this line too long. Please use md5 instead

Reply:
First pr, so not sure what that usually looks like. Replace the string with its md5 equivalent?

Reply:
```suggestion
            'description': 'md5:abcd'
```

replace `abcd` with the md5

Reply:
> First pr, so not sure what that usually looks like. Replace the string with its md5 equivalent?

Yes, it must looks like
```
'description': 'md5:<your_md5_here>'
```
If you didn't have hash checker, you can get the md5 from test, just set `md5:fixme` or another value and then the test give you right md5 value